Documentation/capabilities-guide.md

```diff
@@ -300,6 +300,10 @@ without changing the executed images.
 The `--cap-retain` option to `rkt run` manipulates the `retain` capabilities set.
 The `--cap-remove` option manipulates the `remove` set.
 
+Capabilities specified from the command-line will replace all capability settings in the image manifest.
```

Comment:
They will be overridden. This guide intro also describes that the two isolators are mutually exclusive, and that retain/remove are just two approaches (whitelist/blacklist) to manipulate the same set.